src/rust/ide/view/graph-editor/src/component/node/action_bar.rs

```diff
@@ -87,9 +87,12 @@ impl Icons {
         let freeze         = ToggleButton::new(&logger);
         let visibility     = ToggleButton::new(&logger);
         let skip           = ToggleButton::new(&logger);
-        display_object.add_child(&freeze);
         display_object.add_child(&visibility);
-        display_object.add_child(&skip);
+        // Note: Disabled for https://github.com/enso-org/ide/issues/1397
+        //  Re-enable when they are needed again.
```

Comment:
This comment tells nothing about this change. It just tells that this is "dsiabled" and was disabled as part of "visual improvements" and it should be re-enabled "when needed again". Such comments should be descriptive, in particular - why it is disabled (because these icons were useless now) and what does it mean "needed again" - when icons willbecome implemented properly + it should have link to issue about that (fixing / finishing the icons skip + freeze)

Reply:
Found the related issue and linked it. Expanded the description.